ephios/settings.py

```diff
@@ -114,6 +114,12 @@
 
 DATABASES = {"default": env.db_url()}
 
+# Caches
+CACHES = {"default": env.cache_url(default="locmemcache://")}
+SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
+DJANGO_REDIS_IGNORE_EXCEPTIONS = True
+DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS = True
+
```

Comment:
So what happens if I do a fresh checkout and try to start ephios? Does it fail anyway?

Reply:
if you set up a development environment, no CACHE_URL is set and therefore the default locmemcache is used. You have to actively set the CACHE_URL to point to your redis instance to enable it. If it cant reacht the cache, it will fail silently and just use the database.